docs/data/tool/config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Node dependencies
+ */
+const path = require( 'path' );
+
+const root = path.resolve( __dirname, '../../../' );
+
+module.exports = {
+	namespaces: {
+		core: {
+			title: 'WordPress Core Data',
+			// Figures out a way to generate docs for the dynamic actions/selectors
+			selectors: [ path.resolve( root, 'packages/core-data/src/selectors.js' ) ],
+			actions: [ path.resolve( root, 'packages/core-data/src/actions.js' ) ],
+		},
+		'core/blocks': {
+			title: 'Block Types Data',
+			selectors: [ path.resolve( root, 'blocks/store/selectors.js' ) ],
+			actions: [ path.resolve( root, 'blocks/store/actions.js' ) ],
+		},
+		'core/editor': {
+			title: 'The Editor\'s Data',
```

Comment:
Does the style guide not let us use double quotes rather than escaping? It's sooooo ugly.

Reply:
It looks like this is how things are set up.

Reply:
Filed #7366, then 😄